django/core/management/commands/inspectdb.py

```diff
@@ -255,7 +257,8 @@ def get_meta(self, table_name, constraints):
                 if len(columns) > 1:
                     # we do not want to include the u"" or u'' prefix
                     # so we build the string rather than interpolate the tuple
-                    tup = '(' + ', '.join("'%s'" % c for c in columns) + ')'
+                    tup = '(' + ', '.join(
+                        "'%s'" % column_to_field_name[c] for c in columns) + ')'
```

Comment:
keeping this on a single line is okay (we allow up to 119 characters when it helps readability)